### PR TITLE
Add unit tests for the with_only function

### PR DESCRIPTION
**Pull request by sanjevShakya**
Description: 
- Add unit test for with_only method
Screenshot from 2020-02-18 14-22-03

### CODE REVIEW

**Comment by sanjevShakya** [comment]
@kabirbaidhya 
Note: attribute ```iteritems()``` of a dictionary is deprecated and used as ```items()```

File: tests/util/test_object.py [hunk 1]
```diff
@@ -233,3 +233,68 @@ def test_merge_v2():
         "baz": "Baz",
         "test": {"attr1": "Foo Bar", "attr2": None, "attr3": ["1", "2"]},
     }
+
+
+def test_with_only_returns_dictionary_when_both_src_and_attrs_is_valid_arguments():
```
Comment (praneen) [inline tests/util/test_object.py]: are_valid_arguments?

File: datamegh/util/object.py [hunk 2]
```diff
@@ -46,9 +46,19 @@ def with_only(src, attrs):
     Return a new copy of source dictionary
     containing only the attributes provided.
     """
+    if not is_dict(src):
+        raise AttributeError(
+            "First argument must be a dictionary, invalid argument received '%s'."
+            % (src)
```
Comment (kabirbaidhya) [inline datamegh/util/object.py]: This is the old way of formatting strings. Use `str.format` method always.
```py
"First argument must be a dictionary, invalid argument received '{}'.".format(src)
```

File: datamegh/util/object.py [hunk 3]
```diff
@@ -46,9 +46,19 @@ def with_only(src, attrs):
...
+        )
+    elif not is_list(attrs):
+        raise AttributeError(
+            "Second argument must be a list, invalid argument received '%s'." % (attrs)
```
Comment (kabirbaidhya) [inline datamegh/util/object.py]:
```suggestion
            "Second argument must be a list, invalid argument received '{}'.".format(attrs)
```

**Pull request merged**